.github/workflows/matrix.yml

```diff
@@ -0,0 +1,112 @@
+name: Linting & Integration tests matrix
```

Comment:
Rename, no longer a matrix, also edit filename so this is identity-service checks